docs/paper/paper.md

```diff
@@ -0,0 +1,85 @@
+
+\autoref{fig:fig1} shows an example expedition around the Agulhas Current and South Eastern Atlantic, deploying a suite of instruments to sample physical and biogeochemical properties. Notable oceanographic features, such as the strong Agulhas Current and Agulhas Retroflection (drifters retroflecting back into the Indian Ocean), are clearly visible via the underway ADCP measurements (\autoref{fig:fig1}b) and drifter releases (\autoref{fig:fig1}c), respectively, in the early waypoints. CTD profiles also capture the vertical structure of temperature and oxygen across the expedition route, including the warmer surface waters of the Agulhas region (\autoref{fig:fig1}d, early waypoints) and the Oxygen Minimum Zone in the South Eastern Atlantic (\autoref{fig:fig1}e, final waypoints).
+
+<!-- TODO: may add an example Argo plot here, instead of the CTD (temperature) -->
+<!-- TODO: insert as an .imshow() of the .png exported from the plotly 3D plot -->
+
+![Example VirtualShip expedition simulated in July/August 2023. Expedition waypoints displayed via the NIOZ MFP tool (a), Underway ADCP measurements (b), Surface drifter releases (c; 90-day lifetime per drifter), and CTD vertical profiles for temperature (d) and oxygen (e). Black triangles in b), d) and e) mark waypoint locations across the expedition route, corresponding to the purple markers in a).\label{fig:fig1}](figure1.png)
+
+The software is designed to be highly accessible to the user. It is wrapped into three high-level command line interface commands (using [Click](https://click.palletsprojects.com/en/stable/)):
+
+1. `virtualship init`: Initialises the expedition directory structure and a `expedition.yaml` configuration file, which controls the expedition route, instrument choices and deployment timings. A common workflow is for users to first define expedition waypoint locations via the external [NIOZ Marine Facilities Planning](https://nioz.marinefacilitiesplanning.com/cruiselocationplanning#) (MFP) mapping tool. The coordinates can be exported and fed into `init` via the `--from-mfp` flag.
+2. `virtualship plan`: Launches a user-friendly Terminal-based expedition planning User Interface (UI), built using [`Textual`](https://textual.textualize.io/). This allows users to intuitively modify their expedition waypoint locations, timings and instrument selections.
```

Comment:
They can modify the waypoint locations, but have to set the timings and instrument selection. The wording now suggests that these already come from the MFP site